controllers/numaresourcesoperator_controller.go

```diff
@@ -222,6 +222,11 @@ func (r *NUMAResourcesOperatorReconciler) reconcileResource(ctx context.Context,
 		}
 	}
 
+	// TODO modify the trees to include only rtepause disabled (nil/empty/disabled)
+	// log "RTEsync is disabled for nodegroups ..."
+	// check length of the new trees, if >0 continue with the sync, otherwise return:
+	//		return ctrl.Result{}, status.ConditionAvailable, nil
+
```

Comment:
the code here is a bit more convoluted than I like it to be. You may likely want to look about how we create per-nodeGroup daemonsets when we iterate over `existing.State` around line 410

Reply:
*MAYBE* just change the `daemonsetUpdater` function

Reply:
thanks for your review and comments, I haven't gone deeply into this yet, I'll surely verify you suggestion